EngineeringModel/ViewModels/RuleVerificationListBrowser/RuleVerificationRowViewModel.cs

```diff
@@ -35,6 +38,14 @@ protected RuleVerificationRowViewModel(RuleVerification ruleVerification, ISessi
             : base(ruleVerification, session, containerViewModel)
         {
             this.UpdateProperties();
+
+            //A subscription is made here in addition to the one made in the base class in order to be notified of non-persisntent changes which will not result in an updated revision number
+            var thingSubscription = CDPMessageBus.Current.Listen<ObjectChangedEvent>(this.Thing)
```

Comment:
There is a virtual method `InitializeSubscriptions` in the RowViewModelBase that you could override where these kind of subscriptions can be created.

Reply:
Done